sdk/search/azure-search-documents/src/main/java/com/azure/search/annotation/FieldIgnore.java

```diff
@@ -0,0 +1,14 @@
+package com.azure.search.annotation;
+
+import com.fasterxml.jackson.annotation.JacksonAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldIgnore {
```

Comment:
What about `IgnoreField`? Not only does it read better, but fits word order better (comments below).

Reply:
Mimic Jackson package naming. 
They use @JsonIgnore if they don't want to serialize the property.

Reply:
Do you have any name suggestion?
@IgnoreField ?